dependency-reduced-pom.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This file shouldn't really be here - you can stop it from being created or add it to the `.gitignore`